packages/maskbook/src/components/InjectedComponents/PayloadReplacer.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { Chip, makeStyles, Theme, Typography, Link } from '@material-ui/core'
+import { makeStyles, Theme, Typography, Link, withTheme } from '@material-ui/core'
```

Comment:
Using `withTheme()` rather than `getMaskbookTheme()`, since we are using the the color of former one.